docs/diagrams/MarkdownTextStylerDiagram.puml

```diff
@@ -0,0 +1,41 @@
+@startuml
+!include style.puml
+
+
+box UI UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+end box
+box Logic LOGIC_COLOR_T1
+participant ":Command" as Command LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+box UI UI_COLOR_T1
+participant ":ResultDisplay" as ResultDisplay UI_COLOR
+participant "<<class>>\nTextStyleHelper" as TextStyleHelper UI_COLOR
+end box
```

Comment:
Would it look better if you shift all the UI components left?

Reply:
i thought about it but since I wanted to follow the flow I thought it would better splitting it up

Reply:
Fair choice, we can keep it as this then.